src/raft.rs

```diff
@@ -1323,9 +1323,15 @@ impl<T: Storage> Raft<T> {
                 "last index" => m.reject_hint,
                 "from" => m.from,
                 "index" => m.index,
+                "last term" => m.log_term,
             );
 
             if pr.maybe_decr_to(m.index, m.reject_hint, m.request_snapshot) {
+                let conflict_index = self.raft_log.find_conflict_by_term(pr.next_idx, m.log_term);
```

Comment:
For backward compatibility, please check `m.log_term == 0`.

Reply:
> For backward compatibility, please check `m.log_term == 0`.

I can fix this, but I wonder when does `m.log_term == 0` occur? After become `leader`, the term is at least 1, unless we send message with log_term 0 for testing.

Reply:
Because `log_term` is not filled for `MsgAppendResponse` before this PR, so it will be 0. So if a Raft group is in rolling update, it could become a problem.